hoomd/md/pair/pair.py

```diff
@@ -490,8 +496,9 @@ class Ewald(Pair):
 
     The Ewald potential is designed to be used in conjunction with PPPM.
 
-    See `Pair` for details on how forces are calculated and the available
-    energy shifting and smoothing modes. Use the `params` dictionary to set
+    See `Pair` for details on how forces are calculated. Note Ewald does not
+    support energy shifting or smoothing.
+    Use the `params` dictionary to set
     potential coefficients. The coefficients must be set per unique pair of
     particle types.
```

Comment:
```suggestion
```
Most of the potentials don't have this sentence any longer. Since we are updating them for consistency in this PR, let's remove this from all of them. The requirement is implicit in the definition of the `TypeParameter` type of `params` and it is covered in the tutorial.